### DISCUSSION TITLE
New name for the extension

### DISCUSSION THREAD

**Discussion by OhMyGuus** [OhMyGuus/I-Still-Dont-Care-About-Cookies#23] [Polls]
Let's vote for it 👍

**Comment by OhMyGuus** [comment]
Oops didn't expect adding a option to remove the previous votes  (won't edit anymore so u all can just vote 👍 )

**Comment by mmuman** [comment]
Put that jar out of my sight

**Reply by ryanginnelly252** [reply]
😂

**Comment by winskil** [comment]
#20 (Name, Version, Author update) has been updated to a new name!
![image](https://user-images.githubusercontent.com/44670482/191839419-3d8c3299-551c-4aac-a374-cd34d45d3262.png)

**Comment by elad464** [comment]
You should name it "Cookie Monster"

**Reply by Cwpute** [reply]
already taken by http://www.cookie-monster.io/

**Comment by pineapples5972** [comment]
When the name will be decided?

**Reply by winskil** [reply]
![image](https://user-images.githubusercontent.com/44670482/197477048-e02f827d-db6b-44bc-9b68-c7542b2e9877.png)
Around 28 days ago

**Reply by pineapples5972** [reply]
ohh name decided but still not yet implemnted?

**Reply by winskil** [reply]
It has been implemented already. Just a couple of comments above yours.
<img width="910" alt="image" src="https://user-images.githubusercontent.com/44670482/197504930-780597e7-8b2a-4313-8f67-3c9cfe1b79c7.png">
The extension is already up in Firefox/Chrome/Edge stores with the new name.

**Reply by pineapples5972** [reply]
ohh i thought repo name also get changed.

**Reply by pineapples5972** [reply]
btw this name was first suggested by someone named AmusedAstronaut in matrix room.
and i just put it here because it sounds so appealing and humorous.

**Reply by pineapples5972** [reply]
Just mentioning here just in case if question arise of credit.

**Reply by OhMyGuus** [reply]
> ohh i thought repo name also get changed.

The repo name will be changed as well soon, delayed it a bit because of Google & Github search
